src/Build/Logging/BinaryLogger/BinaryLogger.cs

```diff
@@ -53,7 +53,9 @@ public sealed class BinaryLogger : ILogger
         //   - TargetSkippedEventArgs: added OriginallySucceeded, Condition, EvaluatedCondition
         // version 14:
         //   - TargetSkippedEventArgs: added SkipReason, OriginalBuildEventContext
-        internal const int FileFormatVersion = 14;
+        // version 15:
+        //   - Log only environment variables accessed as properties
```

Comment:
I'd be more specific and mention that we now only log the used env vars on BuildFinishedEventArgs